### ISSUE TITLE
Neural Networks Explained

### ISSUE THREAD

**Issue by maxhelgestad** [Noahhobe/TeamOne2020#8]
https://www.youtube.com/watch?v=GvQwE2OhL8I

This video was a good introduction to neural networks, which I knew nothing about. It goes through step by step to show how a network is constructed and the meaning of each node. It also describes the use of the activation function, learning rate, momentum, iterations, and error level.
